charts/prometheus-pushgateway/templates/service.yaml

```
@@ -19,6 +19,10 @@ spec:
   {{ else if and .Values.runAsStatefulSet (not (has .Values.service.type $stsNoHeadlessSvcTypes)) }}
   clusterIP: None # Headless service
   {{- end }}
+  {{- if .Values.service.ipDualStack.enabled }}
+    ipFamilies: {{ toYaml .Values.service.ipDualStack.ipFamilies | nindent 4 }}
+    ipFamilyPolicy: {{ .Values.service.ipDualStack.ipFamilyPolicy }}
```

Indentation goes too far:

```suggestion
  ipFamilies: {{ toYaml .Values.service.ipDualStack.ipFamilies | nindent 4 }}
  ipFamilyPolicy: {{ .Values.service.ipDualStack.ipFamilyPolicy }}
```

---

Didn't see it on my IDE, good catch 👍